packages/renderer/src/validate-ffmpeg.ts

```diff
@@ -94,8 +141,6 @@ export const validateFfmpeg = async (
 			);
 		}
 
-		throw new Error('FFmpeg not found');
+		process.exit(1);
```

Comment:
revert this (process.exit will quit the whole server that one might be running, without any chance to try/catch {} the error)